apps/cms/lib/page/event.ex

```diff
@@ -96,9 +106,38 @@ defmodule CMS.Page.Event do
     }
   end
 
-  @spec past?(t, Date.t()) :: boolean
-  def past?(event, now) do
-    Date.compare(event.start_time, now) == :lt
+  @spec started_status(
+          NaiveDateTime.t() | DateTime.t(),
+          NaiveDateTime.t() | DateTime.t() | nil
+        ) :: status
+  # Events have DateTime start/ends.  Teasers have NaiveDateTimes.
+
+  def started_status(%NaiveDateTime{} = start, stop) do
+    started_status(
+      convert_using_timezone(start, ""),
+      if(is_nil(stop), do: nil, else: convert_using_timezone(stop, ""))
+    )
+  end
+
+  def started_status(start, nil) do
+    cond do
+      time_is_greater_or_equal?(now(), start) and Date.compare(now(), start) === :gt -> :ended
+      time_is_greater_or_equal?(now(), start) -> :started
+      true -> :not_started
+    end
+  end
+
+  def started_status(start, stop) do
+    cond do
+      time_is_greater_or_equal?(now(), start) and !time_is_greater_or_equal?(now(), stop) ->
+        :started
+
+      time_is_greater_or_equal?(now(), stop) ->
+        :ended
+
```

Comment:
Since the conditions are evaluated in order, I wonder if it makes sense to first look for ended, and then started, e.g.

```elixir
cond do
  time_is_greater_or_equal?(now(), stop) -> :ended
  time_is_greater_or_equal?(now(), start) -> :started
  true -> :not_started
end
```